code/__DEFINES/say.dm

```diff
@@ -45,6 +45,19 @@
 #define MODE_ALIEN "alientalk"
 #define MODE_HOLOPAD "holopad"
 
+#define MODE_CHANGELING "changeling"
+#define MODE_KEY_CHANGELING "g"
+#define MODE_TOKEN_CHANGELING ":g"
+
+// ling status, their interaction with the hivemind
+
+///not in the hive
+#define LINGHIVE_NONE 0
+///lost their powers, gets a special message
+#define LINGHIVE_FALLEN 1
```

Comment:
I don't see any way for this to ever actually be returned by `lingcheck()`, am I missing something?

Reply:
my bad, will include